src/vs/workbench/contrib/files/browser/views/explorerViewer.ts

```diff
@@ -528,13 +528,13 @@ export class FilesRenderer implements ICompressibleTreeRenderer<ExplorerItem, Fu
 	}
 }
 
-interface CachedParsedExpression {
+export interface CachedParsedExpression {
```

Comment:
is this export used?

Reply:
Not anymore if the ResourceGlobMatcher is used instead.

Reply:
If the export isnt used then we should just keep it internal